spirv_msl.cpp

```diff
@@ -1232,12 +1288,15 @@ string CompilerMSL::entry_point_args(bool append_comma)
 			auto &type = get<SPIRType>(var.basetype);
 
 			if ((var.storage == StorageClassUniform || var.storage == StorageClassUniformConstant ||
-			     var.storage == StorageClassPushConstant) &&
-			    !is_hidden_variable(var))
```

Comment:
Because `storage_class_is_interface` returns true for `StorageClassUniformConstant`. Since uniform storage is a hidden variable, it won’t be added as an main function arguments.